src/core/permissions/Permission.js

```diff
@@ -13,23 +13,17 @@ function enumFactory(friendlyName) {
   };
 }
 
-export class Permission extends Enum {}
+export class Permission extends Enum {
```

Comment:
+1, I think this enum is going to disappear when we use `loom-data`, and when you start working on organizations.